### DISCUSSION TITLE
Project Request - V-shaped Interior Permanent Magnet Synchronous Machine

### DISCUSSION THREAD

**Discussion by bcaldeirar764** [Eomys/pyleecan#700] [Q&A]
Hello everyone,

I'm in search of a complete project for a V-shaped IPMSM with concentrated winding and with power greater than 10kW. I know that Pyleecan has a base file, but since I work with analytical modeling rather than machine design, I believe it would be more appropriate to use a well-established project. Does anyone know if there is another base example in Pyleecan or if any of you would know where to find one?

I have complete data for similar machines, but not specifically for the one I need.

Thank you very much, and I apologize if this is not the appropriate place for this request.

**Comment by BonneelP** [comment]
Hello,

Thank you for your interest in pyleecan. I'm not sure if I correctly understand your question here are few elements/questions:
- Pyleecan is able to simulate IPMSM with several layers of magnets whatever their configuration (V shape or other) with any winding (we have the star of slot available in the GUI but user defined is possible too). The simulation is current driven sinusoidal and can be at variable speed. The computation of the flux can be done with FEMM and the mesh and the solution are available for losses computation for instance
- Regarding the "base example" are you looking for the tutorials ? https://pyleecan.org/tutorials.html. There are also several "demo machine" within pyleecan that you can use
- Can you further detail what is your need and your aim ? Are you trying to compare pyleecan results with experimental data ?

Best regards,
Pierre

**Reply by bcaldeirar764** [reply]
Hello, Pierre

Thank you for your response!

I may have expressed myself poorly. I always use the graphical interface of Pyleecan, so please keep that in mind. What I do is initially select the type of machine, which in my case is "IPMSM", and then a folder with several .json files is shown. I open all of them, but none of them is the machine I need. The Toyota Prius one is the closest; however, the stator winding is distributed, and I necessarily need it to be concentrated.

I know that Pyleecan allows adjusting and even generating the machine with the characteristics I need. However, since I don't have a ready-made project on hand and I am not from this field, I would like to know if Pyleecan has a base file for this type of machine that is already well-designed. This way, I could look up the nominal operating data on the internet, in case it is a machine like a known car motor. Is it a bit clearer now?

Thank you very much for your attention!
